welcome.md

```diff
@@ -0,0 +1,110 @@
+# Welcome to our office
+Do you really want to be a software developer? It means painstaking hours of staring at commas, indentation and semicolons only to find out you've made a small typo. 
+
+This guide covers a few topics:
+* [Introduction](#introduction)
+* [Tools](#tools)
+* [Learning JavaScript](#learning-javascript)
+* [Learning Python](#python)
+
+
+# Introduction
+Here at Nelen & Schuurmans we use a number of tools and language that we deem a good fit for our company, namely: Python and JavaScript.
+
+Why? The list of things you can do with Python/JavaScript is endless:
```

Comment:
In het lijstje staan veel dingen genoemd die we bij N&S niet gebruiken. Reclame voor python, hoe hard is dat nodig? Het is gewoon een gegeven dat we het gebruiken. Idem javascript. Niet-gebruikte zaken zoals micropython en phaser lijken me verwarrend in dit algemeen-geldige WOM document.